ibis/__init__.py

```diff
@@ -125,3 +128,18 @@ def connect(*args, **kwargs):
         setattr(proxy, name, getattr(backend, name))
 
     return proxy
+
+
+def __getattr__(name: str) -> Any:
```

Comment:
Since we already had a `__getattr__` I went this route to keep the typing for the `load_backend` portion. Once we deprecate NA completely we can remove the workaround